packages/mystmd/tests/basic-tex-math/input.md

```diff
@@ -4,6 +4,12 @@ Paragraph
 $$Ax=b$$
 Paragraph
 
+Paragraph around dollar math
+$$
+Ax=b
+$$
+Paragraph
+
```

Comment:
@dellaert can you confirm that this is the output that you would expect (see the outcome right below).

Reply:
Yep!